mysys/stacktrace.c

```diff
@@ -185,7 +185,7 @@ static int print_with_addr_resolve(void **addrs, int n)
   int i;
   const char *err;
 
-  if ((err= my_addr_resolve_init()))
+  if ((err= my_addr_resolve_init(addrs[0])))
```

Comment:
If `n==0`, then this will SIGSEGV in a way that it never would have before, because you're unconditionally dereferencing `addrs[0]`.

Consider adding…
```diff
+if (n < 1)
+  return 1;    // Nothing to do
+
 if ((err= my_addr_resolve_init(addrs[0])))
 {
```

Reply:
n==0 means backtrace() returns zero (or NULL), which implies the program has no stack (or there is something broken in libcX).

Reply:
In the corner cases where the stack trace can't be generated (we've seen a few :man_shrugging:), it'll help a lot to not generate a new `SIGSEGV` as a result.